### DISCUSSION TITLE
Where do I get the pv_forecast data from?

### DISCUSSION THREAD

**Discussion by C0re568** [Akkudoktor-EOS/EOS#168] [Q&A]
Yesterday I got the eso server up and running (with some fixes) with my data. The only thing I do not really have is the pv_forecast. How do you calculate the forecast? Do you use a public api for this?

**Comment by NormannK** [comment]
you need an api. I suggest to use forecast.solar.

Example link:
https://api.forecast.solar/estimate/watt_hours_period/55.5967356/10.835667/45/90/4

you are limited in the free api calls so pay attention while writing something on your own.

Documentation:
https://doc.forecast.solar/api

**Comment by drbacke** [comment]
We coded also an API, but it's yet Not Open. 
You can find IT Here API.akkudoktor.net
Please Just use it for dev
